c2cgeoportal/scaffolds/update/CONST_vars.yaml_tmpl

```diff
@@ -9,7 +9,7 @@ vars:
     srid: -1
 
     # the Unix user under which the modwsgi daemon processes are executed.
-    modwsgi_user: www-data
+    modwsgi_user: MODWSGI_USER
```

Comment:
Your are not supposed to add something to interpreted/environment for that to work?